doc/conf.py

```diff
@@ -26,11 +27,20 @@
 # Add the 'extensions' directory to sys.path, to enable finding Sphinx
 # extensions within.
 sys.path.insert(0, os.path.join(ZEPHYR_BASE, 'doc', 'extensions'))
-# Also add west, to be able to pull in its API docs.
-sys.path.append(os.path.join(ZEPHYR_BASE, 'scripts', 'meta'))
-# HACK: also add the runners module, to work around some import issues
-# related to west's current packaging.
-sys.path.append(os.path.join(ZEPHYR_BASE, 'scripts', 'meta', 'west'))
+
+try:
+    desc = check_output(['west', 'list', '-f{abspath}', 'west'],
+			stderr=DEVNULL,
+			cwd=os.path.dirname(__file__))
+    west_path = desc.decode(sys.getdefaultencoding()).strip()
+    # Add west, to be able to pull in its API docs.
+    sys.path.append(os.path.join(west_path, 'src'))
+except FileNotFoundError as e:
+    raise RuntimeError('Unable to find west. Make sure you have installed'
```

Comment:
missing spaces at end of each string. This would read as 'installedit' and the next lines would result in 'installationusing'.